tests/DirectionProvider_test.jsx

```diff
@@ -40,7 +49,7 @@ describe('<DirectionProvider>', () => {
   it('broadcasts the direction when the direction prop changes', () => {
     const direction = DIRECTIONS.LTR;
     const nextDirection = DIRECTIONS.RTL;
-    const wrapper = shallow(
+    const wrapper = mount(
```

Comment:
please do not alter these tests; these must remain with shallow, and new tests added to cover these changes.

Reply:
There is still an open issue on Enzyme being incapable of causing cDU to fire when `setProps` is called on the wrapper. The _only_ known workaround is using `mount` over `shallow`. This change is isolated to this file, and only tests that need to ensure this behavior is working correctly.

Reply:
Can you link to that enzyme issue?

tbh I'd rather wait for that to be resolved.

Reply:
Meant to include it in my notes above. Here it is (I'll add it above as well): https://github.com/airbnb/enzyme/issues/1452